### DISCUSSION TITLE
Listing compiled files at sources panel of developer tools

### DISCUSSION THREAD

**Discussion by korc** [FranckFreiburger/vue3-sfc-loader#114] [Q&A]
When running vue app in development mode using `yarn serve`, you can see all `.vue` files listed in sources panel of the browser's developer tools interface, under `webpack://`.
This is extremely convenient for debugging components during development process.
Any way to get similar functionality with vue3-sfc-loader?
(FYI, using only vue2 for now)

**Comment by pac0rro** [comment]
Hi,

I love this library, but for me this is an stopper. I can't live without the debugger.

The only fix I tested was to put the [debugger keyword](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/debugger) in the javascript code, but its a little dirty because debugger shows up the digested javascript and only works "well" in Chrome.
